env.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
This script looks suspiciously like `envsubst` but I guess its not installed in the nginx container!

Reply:
Haven't used this command before -thanks! It does look very similar - I suppose the difference is that during the build process, the globally injected import.meta.env.VITE_API_URL variable is statically replaced with a string (in our case 'MY_APP_API_URL') rather than an environment variable, as the env variable is not yet known. This means that in our docker entrypoint script, we can do a sed to find and replace the string with our desired value (which we inject as an env var). Let me know if I've misunderstood the use case/usage?